projects/cmake/CMakeLists.txt

```diff
@@ -66,6 +66,8 @@ SET(EXAMPLES_SOURCES
   ${ROOT_DIR}/kits/arm/gravity_compensation.cpp
   ${ROOT_DIR}/kits/arm/teach_repeat.cpp)
 
+find_package(PythonLibs 3.6)
```

Comment:
I need to check to see what this does on newer versions of python (_e.g._, 3.7). I think it should be ok, but just want to make sure.

Also, more of a note to myself, but we need to document that you need to install `python3-dev`/`python3-devel` to get the headers for Python.